jenkins/bootstrap.py

```diff
@@ -718,7 +720,7 @@ def setup_root(call, root, repo, branch, pull, ssh):
 def bootstrap(
     job, repo, branch, pull, root, upload, robot, timeout=0, use_json=False, ssh=False):
     """Clone repo at pull/branch into root and run job script."""
-    # pylint: disable=too-many-locals,too-many-branches
+    # pylint: disable=too-many-locals,too-many-branches,too-many-statements
```

Comment:
this might be a sign this method needs to be broken up... (but don't need to do that here)